mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_2248 net/minecraft/block/Block
+	COMMENT A block type.
```

Comment:
I'd argue we don't really need javadoc in this capacity since this is a pretty universally known concept.

Reply:
If these do have documentation, it's probably worth mentioning that they are singleton instances. Especially for items, it's a common mistake to attach state to an item implementation which becomes unintentionally global.

Reply:
That's not what a singleton is.